### PR TITLE
Adding Preact into Riffy Blog

### PR DESCRIPTION
**Pull request by rayriffy**
This gonna be lit 🔥🔥🔥

### CODE REVIEW

**Comment by aomkirby123** [comment]
It’s alright, you should wait until it’s stable to see if this fixed the problem, also I forgot to mention: according to your PR of updating from 8 to x, Preact X has build in for Preact compat for using with Preact X. Using Preact X with old Preact compat might be the reason why dangerouslySetInnerHTML break. Also Preact Compat X does has it own createContext, when using these feature you might want to alias Preact/compat to React-DOM instead (or include this instead of React-DOM)
Also when using JSX in Preact, you have to include { h } like how you included React just in case something is wrong.
Long story short, I might try solving this problem and created PR when I had some spare time.

**Comment by rayriffy** [comment]
Okay, just create PR merging to `master` to get all available deploy preview.

**Comment by rayriffy** [comment]
Bad news, `gatsby-plugin-preact` already defined `preact/compact`

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-preact/src/gatsby-node.js
